### ISSUE TITLE
Vampire Bat Form Works in Defficiency Goonecode Satellite

### ISSUE THREAD

**Issue by Shadowmech88** [d3athrow/vgstation13#8342]
Apparently it should not be able to.

**Comment by Probe1** [comment]
Issue is still present.  No idea how to fix it.

**Comment by Shadowmech88** [comment]
I believe there's some kind of flag that can be applied to areas to prevent things like jaunting, isn't there?

**Comment by Probe1** [comment]
Blessed tiles have NOJAUNT.

**Comment by Probe1** [comment]
.. which none of the map tiles have.

**Comment by Probe1** [comment]
Yeah so after testing, every map allows jaunting on the goon sat.

**Comment by Kurfursten** [comment]
@Shadowmech88 This is not a bug apparently, so you can close it

**Comment by Shadowmech88** [comment]
How do you know it's not a bug?

**Comment by Kurfursten** [comment]
Probe says this works on every map

I guess it could be a feature request though. But... is there really a reason we need to stop goonejaunting...?

**Comment by Shadowmech88** [comment]
I can't remember the other satellites well enough, but I know that the Defficiency satellite has a big loot room at the end.
Allowing vampires/wizards to jaunt on the satellite kind of just guarantees them all that loot for no work, doesn't it?

**Comment by Kurfursten** [comment]
Anyone can get into that room by just deconstructing one (1) reinforced wall. In fact, you pretty much have to, because the emitter maze was designed for our old beamcode from 2 years ago where it would pulse instead of a constant stream of low damage. In other words, you can't get through without getting mangled unless you cheese it by building lockers to stuff in front of the emitters.

Kinda makes you think, doesn't it?

**Comment by Shadowmech88** [comment]
I suppose if it's that easy to get in there, the jaunting isn't a concern.
Just to be certain, are you sure it's not a super reinforced wall?

**Comment by Kurfursten** [comment]
That is a good question, I can't compile my local right now to check though

**Comment by Duny-** [comment]
1/The maze was designed with continuous beams in mind, cheesing it by building things is perfectly fine
2/The walls are not deconstructable
3/The entire goonsat is supposed to prevent every kind of teleportation, I added snowflake code to the area to prevent hand teles and wizard jaunt/blink from working if I remember right.
4/I don't know how to prevent vampires from doing it, please send help.
